src/data-pipeline/parameter.ts

```diff
@@ -135,6 +135,17 @@ export function createStackParameters(scope: Construct) {
     type: 'Number',
   });
 
+  const emrApplicationArchitectureParam = new CfnParameter(scope, 'EmrApplicationArchitecture', {
+    description: 'Emr-Serverless application architecture',
+    default: 'Auto',
+    allowedValues: [
+      'Auto',
```

Comment:
Let's move those as constants to share with stack.

Also, could you reuse those architect definitions of the AWS SDK?

Reply:
changed.